tracing-appender/src/rolling/builder.rs

```diff
@@ -271,3 +274,15 @@ impl Default for Builder {
         Self::new()
     }
 }
+
+#[cfg(test)]
+mod test {
```

Comment:
nit: normally we name modules containing tests "tests" rather than "test":
```suggestion
mod tests {
```